diff --git a/dist/leva.esm.js b/dist/leva.esm.js
index d3b68a488f98a26819fc4b92bf656b0373806535..b5be8e7ec9cf665d3e90531507504c52ccda8a3a 100644
--- a/dist/leva.esm.js
+++ b/dist/leva.esm.js
@@ -5,6 +5,7 @@ import { extend, colord, getFormat } from 'colord';
 import namesPlugin from 'colord/plugins/names';
 import { dequal } from 'dequal/lite';
 import 'react-dom';
+import { createRoot } from 'react-dom/client'
 import React, { useRef, useMemo, useLayoutEffect, useEffect, useState, useCallback, forwardRef, useImperativeHandle } from 'react';
 import { RgbaColorPicker, RgbColorPicker } from 'react-colorful';
 import shallow from 'zustand/shallow';
@@ -2169,9 +2170,9 @@ function useRenderRoot(isGlobalPanel) {
         });
         if (document.body) {
           document.body.appendChild(rootEl);
-          render(React.createElement(Leva, {
+          createRoot(rootEl).render(React.createElement(Leva, {
             isRoot: true
-          }), rootEl);
+          }));
         }
       }
       rootInitialized = true;
